.github/workflows/continuous-integration.yml

```diff
@@ -672,11 +672,11 @@ jobs:
           BUILDTYPE: vagovprod
           NODE_ENV: production
 
-  testing-reports:
-    name: Testing Reports
+  failure-notifications:
```

Comment:
With this no longer being part of CI, it might be good to add `concurrency` to testing-report.yml

It will prob be a good idea to do a in a separate PR, but just shedding some light on it